axelrod/strategies/meta.py

```diff
@@ -1,4 +1,5 @@
 from numpy.random import choice
+from random import random,choice
```

Comment:
This is overwriting the `choice` imported in `numpy.random` and causing numerous failures of other tests.

I suggest importing `import random` and using `random.random` but using numpy's `choice` (which has similar behaviour) - alternatively you could use `random.choice`.